inject.go

```diff
@@ -183,7 +183,7 @@ func (i *injector) Get(t reflect.Type) reflect.Value {
 		}
 	}
 	if len(impls) > 1 && i.options.PanicOnAmbiguity {
-		panic(fmt.Sprintf("Expect single matching implementation but found %v", len(impls)))
+		panic(fmt.Sprintf("Expected single matching implementation for type <%v> but found %v: %v", t, len(impls), impls))
```

Comment:
Hmm not sure why fmt package is not doing what it says, but from what I see, Value is printed as <T Value> where T is v's type. So that suggests fmt package is calling String() method?